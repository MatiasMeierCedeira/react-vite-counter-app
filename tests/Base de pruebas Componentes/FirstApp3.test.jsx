import { render,screen } from "@testing-library/react"
import { CounterApp } from "../../src/CounterApp";


    describe('Pruebas First App 3',()=>
        {

          const value =10; 

           test('Should make a match wiht the snapshot',()=>
            {
             const {container} = render(<CounterApp Value = {value}/>);
             expect(container).toMatchSnapshot()
          });

          test('Should be hundred',()=>
            {
             render(<CounterApp Value = {100}/>);
             expect(screen.getByText(100).innerHTML).toBeTruthy();
          });

        });