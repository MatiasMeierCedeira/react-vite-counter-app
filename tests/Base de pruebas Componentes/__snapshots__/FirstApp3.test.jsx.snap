// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas First App 3 Should make a match wiht the snapshot 1`] = `
<div>
  <h1>
    CounterApp
  </h1>
  <h2>
     
     
  </h2>
  <button>
     +1 
  </button>
  <button>
     -1 
  </button>
  <button
    aria-label="btn-reset"
  >
     Reset 
  </button>
</div>
`;
